app/views/catalog/_cocina_default.html.erb

```diff
@@ -6,10 +6,15 @@
   </h2>
   <div id="document-cocina-section" class="accordion-collapse collapse" aria-labelledby="document-cocina-heading">
     <div class="accordion-body" data-controller="json-renderer">
+      <div class="argo-show-btn-group">
+        <a class="btn btn-primary " data-action="json-renderer#expand">Expand</a>
```

Comment:
Could these be `btn-secondary` or `btn-outline-primary`?  I don't really think of collapse expand as primary functions.

Reply:
Sure. That's why I asked you about it yesterday. 😆 I'll tweak 'em.

Reply:
Done!